src/validation.cpp

```diff
@@ -20,6 +20,7 @@
 #include "init.h"
 #include "policy/fees.h"
 #include "policy/policy.h"
+#include "policy/rbf.h"
```

Comment:
Well it would be wrong to rely on a policy constant if ATMP was a consensus critical function, but you're right.